cmd/bench/cmd/root.go

```diff
@@ -0,0 +1,39 @@
+// Copyright Contributors to the Mir project
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	id      string
+	nrNodes int
+	verbose bool
+
+	rootCmd = &cobra.Command{
+		Use:   "bench",
+		Short: "Mir benchmarking tool",
+		Long: `Mir benchmarking tool can run a Mir nodes, measuring latency and
+throughput. The tool can also generate and submits requests to the Mir
+cluster at a specified rate.`,
```

Comment:
```suggestion
		Long: "Mir benchmarking tool can run a Mir nodes, measuring latency " +
			"and throughput. The tool can also generate and submit requests " +
			"to the Mir cluster at a specified rate.",
```

Assuming you don't want the output to be word-wrapped.

Reply:
It's still not perfectly indented in the source code, though looks a little better